Automattic/wp-calypso
# Account Recovery: Add the reset password is succeeded page

## Summary
This PR is based on #13362 and adds the final "reset password is succeeded" page:
![2017-04-27 11 42 19](https://cloud.githubusercontent.com/assets/1842898/25467097/cd5d7710-2b3e-11e7-8921-f565e555a30f.png)

## Test Plan
Unfortunately there is no quick access to this page, so we would need to modify one line of code here.

1. Open [client/account-recovery/controller.js](https://github.com/Automattic/wp-calypso/blob/master/client/account-recovery/controller.js#L16), find the line 16 and update it as:
```
firstStep={ STEPS.RESET_PASSWORD_SUCCEEDED }
```
1. Save and `make run`
1. Visit http://calypso.localhost:3000/account-recovery , and you should see the "Reset password is succeeded" page as the screenshot above.
1. Click "Log in" button should lead you to the login page.
